lib/cloudinary/utils.rb

```diff
@@ -1132,4 +1133,15 @@ def self.patch_fetch_format(options={})
   def self.is_remote?(url)
     REMOTE_URL_REGEX === url
   end
+
+  def self.compute_signature(message, long_url_signature)
```

Comment:
please add a function documentation, including description of parameters and return value